lib/rucio/web/rest/flaskapi/v1/dirac.py

```diff
@@ -55,8 +58,11 @@ def post(self):
         lfns = param_get(parameters, 'lfns')
         ignore_availability = param_get(parameters, 'ignore_availability', default=False)
 
+        LOGGER = logging.getLogger('Dirac.addFiles')
+        LOGGER.setLevel(logging.INFO)
```

Comment:
Why this logger?

Reply:
Oh, sorry, this is a left-over form some testing. Removed.